webapp/components/search_bar.jsx

```diff
@@ -141,6 +141,8 @@ export default class SearchBar extends React.Component {
     handleSubmit(e) {
         e.preventDefault();
         this.performSearch(this.state.searchTerm.trim());
+        $(ReactDOM.findDOMNode(this.refs.search)).find('input').blur();
```

Comment:
@enahum doesn't work with $(this.refs.search).
Also the return of both things are different.
$(this.refs.search) returns me `[SuggestionBox]`, which isn't what I want since I think it's not a DOM element.
Whereas
`$(ReactDOM.findDOMNode(this.refs.search))` returns me `[div]`, which works.